cleanlab_studio/studio/inference.py

```diff
@@ -0,0 +1,101 @@
+import abc
+import csv
+import functools
+import io
+import time
+from typing import List, Union, Optional
+
+import numpy as np
+import numpy.typing as npt
+import pandas as pd
+
+from cleanlab_studio.errors import APIError
+from cleanlab_studio.internal.api import api
+
+
+TextBatch = Union[List[str], npt.NDArray[np.str_], pd.Series]
```

Comment:
You should be able to just use `TextBatch = Iterable[str]` here. TextBatch just needs to be some collection of strings you can iterate through right?

Reply:
True.. although I think Ryan and Anish decided on the user interface, so I'm not sure if we want to support any iterable here.

Reply:
https://www.notion.so/cleanlab/Model-Inference-Python-API-d5209e022f744298835010dbbfb26a4c